app/views/discovery_rules/index.html.erb

```diff
@@ -1,7 +1,7 @@
 <% title _("Discovery Rules") %>
 <% title_actions new_link(_("Create Rule")), help_button %>
 
-<table class="<%= table_css_classes('table-two-pane') %>">
+<table class="<%= table_css_classes('table-fixed') %>">
```

Comment:
Can you please explain why/what is that change and provide some screenshots of before and after?